spec/PushController.spec.js

```diff
@@ -28,7 +28,7 @@ const successfulIOS = function(body, installations) {
   return Promise.all(promises);
 }
 
-describe('PushController', () => {
+fdescribe('PushController', () => {
```

Comment:
I think you forgot to remove this `f` :P

Reply:
Seems to be my super power.